packages/react-native-usb/android/src/main/java/io/trezor/rnusb/ReactNativeUsbBroadcastReceiver.kt

```
@@ -0,0 +1,51 @@
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.hardware.usb.UsbManager
+import android.hardware.usb.UsbDevice
+import android.os.Build
+import android.util.Log
+
+typealias OnUsbDeviceAttachedCallback = (UsbDevice) -> Unit
+typealias OnUsbDeviceDetachedCallback = (deviceName: String) -> Unit
+
+class ReactNativeUsbBroadcastReceiver() : BroadcastReceiver() {
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)